addons/bestja_estimation_reports/models.py

```diff
@@ -0,0 +1,223 @@
+        'bestja.estimation_report',
+        ondelete='cascade',
+        required=True,
+        string="raport szacunkowy",
+    )
+    day_in_store = fields.Many2one('bestja_stores.day', required=True, string="dzień")
+    day = fields.Date(
+        related='day_in_store.date',
+        store=True,
+    )
+    tonnage = fields.Float(required=True, string="Liczba kg")
+    store = fields.Many2one(
+        required=True,
+        related='day_in_store.store.store',
+        store=True,
+        string="sklep",
+    )
+    store_project = fields.Many2one(
+        'bestja.project',
+        required=True,
+        related='estimation_report.project'
+    )
+    store_name = fields.Char(
+        string="nazwa sklepu",
+        related='day_in_store.store.store.display_name',
+        store=True,
+    )
+    store_address = fields.Char(
+        required=True,
+        related='store.address',
+    )
+    store_city = fields.Char(required=True, related='store.city')
+    chain = fields.Many2one(
+        related='store.chain',
+        store=True,
+    )
+    organization = fields.Many2one(
+        'organization',
+        string="organizacja partnerska",
+        related='estimation_report.organization',
+        store=True,
+    )
+    responsible_organization = fields.Many2one(
+        'organization',
+        string="odpowiedzialna organizacja",
+        store=True,  # Needed by graph view
+        related='estimation_report.responsible_organization',
+    )
+
+
+class EstimationReport(models.Model):
+    _name = 'bestja.estimation_report'
+    _inherit = [
+        'protected_fields.mixin',
+        'ir.needaction_mixin',
+    ]
+    _protected_fields = ['state']
+    STATES = [
+        ('sent', 'dodany do zestawienia'),
+        ('draft', 'szkic'),
+    ]
+    project = fields.Many2one(
+        'bestja.project',
+        required=True,
+        string="projekt",
+        domain=lambda self: [
+            ('use_estimation_reports', '=', True),
+            '|',  # noqa
+                ('manager', '=', self.env.uid),
+                ('organization.coordinator', '=', self.env.uid),
+        ],
+        ondelete='restrict',
+    )
+    date = fields.Date(
+        required=True,
+        string="dzień",
+    )
+    name = fields.Char(string="nazwa projektu", related="project.name")
+    report_entries = fields.One2many(
+        'bestja.estimation_report_entry',
+        inverse_name="estimation_report",
+        string="sklep"
+    )
+    state = fields.Selection(STATES, default='draft', string="status")
+    organization = fields.Many2one(
+        'organization',
+        string="organizacja",
+        related='project.organization',
+    )
+    parent_project = fields.Many2one(
+        'bestja.project',
+        string="projekt nadrzędny",
+        related='project.parent',
+        store=True,
+    )
+    responsible_organization = fields.Many2one(
+        'organization',
+        string="zainteresowane organizacje",
```

Comment:
There is another field with the same name in `EstimationReportEntry` model, but descriptions are different…